### PR TITLE
chezmoi v1.8.0 (new formula)

### PR DESCRIPTION
**Pull request by stephengroat**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/chezmoi.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    cd "src/github.com/twpayne/chezmoi" do
+      system "go", "build", "-o", bin/"chezmoi", "main.go"
+
+      # Build bash completion
+      system "make", "completions"
+      bash_completion.install "completions/chezmoi-completion.bash"
+      zsh_completion.install "completions/chezmoi.zsh"
+
+      prefix.install_metafiles
+    end
```
Comment (bayandin) [inline Formula/chezmoi.rb:23]:
```suggestion
    system "go", "build", *std_go_args, "-ldflags", "-s -w"

    system "make", "completions"
    bash_completion.install "completions/chezmoi-completion.bash"
    zsh_completion.install "completions/chezmoi.zsh"

    prefix.install_metafiles
```

**Comment by bayandin** [comment]
Could you please squash commits into one (with a message `chezmoi v1.8.0 (new formula)`), and then we're good to go

**Comment by stephengroat** [comment]
@bayandin squashed in 87571b81235d86ff9eed91e7dc279357e6b44f86

**Comment by bayandin** [comment]
Thanks @stephengroat for your contribution to Homebrew! 🎉 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**